### DISCUSSION TITLE
Welcome to container-blueprints Discussions!

### DISCUSSION THREAD

**Discussion by bikramgupta** [digitalocean/container-blueprints#29] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jimtyrro** [comment]
Question. Current tutorial fails at the Wordpress install using helm with the following error.

`Error: release wordpress failed, and has been uninstalled due to atomic being set: timed out waiting for the condition`

What could be the reason?

**Reply by v-ctiutiu** [reply]
Hi @jimtyrro 

Sorry for the late response. Helm CLI provides a debug flag which makes the install process output more verbose. Theoretically, it should help identify the main issue with the installation process that fails. 

So, all you have to do is append the `--debug` flag to the helm install command for the Wordpress chart. The output can be pretty verbose but you should be able to identify the relevant lines containing the error messages and paste here (if sensitive information is present, don't forget to remove it first).

Thanks.

C.C. @v-bpastiu
